i18n/it/cosmic_settings.ftl

```diff
@@ -27,212 +27,216 @@ import = Importa
 light = Chiaro
 mode-and-colors = Modalità e colori
 recent-colors = Colori recenti
-reset-default = Reimposta predefinito
-reset-to-default = Reimposta predefinito
+reset-default = Ripristina predefinito
+reset-to-default = Ripristina predefinito
 rgb = RGB
-window-hint-accent = Colore di suggerimento della finestra attiva
-window-hint-accent-toggle = Usa il colore d'accento del tema come suggerimento della finestra attiva
+window-hint-accent = Colore d'accento per la finestra attiva
```

Comment:
"Suggerimento" is best fit for "hint".
The border is not only a hint for the user, it's a visual thing that you could enable just because you like it.

Who cares about what window is active when you could have a protocol that follows the mouse (of you can just click in the window, common thing before interacting with it) and focus what the pointer is touching?